assets/js/accredible-dialog.js

```diff
@@ -37,9 +37,37 @@ jQuery( function(){
             };
             
             if (element.tagName === 'A') {
-                const href = element.href;
+                const actionParams = jQuery(element).data('accredibleActionParams');
+                const closeDialog = function() {
+                    dialog.dialog('close');
+                }
                 options.buttons[1].click = function() {
-                    window.location.href = href;
+                    const formData = {};
+                    actionParams.split('&').reduce(function(acc, curr) {
+                        const keyValue = curr.split('=');
+                        acc[keyValue[0]] = keyValue[1];
+                        return acc;
+                    }, formData);
+
+                    // call BE
+                    accredibleAjax.doAutoIssuanceAction(
+                        formData
+                    ).always(function(res){
+                        if ((typeof(res) === 'object')) {
+                            const message = res.data && res.data.message ? res.data.message : res.data;
+                            if (res.success) {
+                                closeDialog();
+                                accredibleToast.success(message, 5000);
+                                // Reload auto issuances.
+                                accredibleAjax.loadAutoIssuanceListInfo().always(function(res){
```

Comment:
I was wondering if `accredibleAjax.loadAutoIssuanceListInfo` is properly handling the pagination.

I updated all page size as `1` and tried to delete one AutoIssuance on page 3. After deleting it, I was expecting the blank third page but the first page was displayed as attached below:

![Screen Shot 2022-06-20 at 10 39 04](https://user-images.githubusercontent.com/20009662/174561638-8fe334d5-16c0-43f3-a2a7-e0d204fe8400.png)

Reply:
Good catch. I should be passing `page_num` with this request.